src/data/history.ts

```diff
@@ -200,6 +200,15 @@ const processLineChartEntities = (
   };
 };
 
+const isNumberValued = (states: HassEntity[]): boolean => {
+  for (const state of states) {
```

Comment:
This is kinda weird handling, what if a state was unavailable in between?

Reply:
You're right. It makes sense to have a special check for `unavailable`.

Reply:
Updated such that:
- If an entity has been unavailable or unknown through all of the displayed history, it's not considered numerical valued.
- Otherwise, unavailable and unknown states are ignored